src/features/shared/get-contactmomenten-service.ts

```diff
@@ -122,5 +124,59 @@ export function useContactmomentenByKlantId(
     url.searchParams.set("klant.id", id.value);
     return url.toString();
   }
-  return ServiceResult.fromFetcher(getUrl, fetchContactmomenten);
+  return ServiceResult.fromFetcher(getUrl, fetchContactmomenten, {
+    getUniqueId() {
+      return getUrl() + "contactmoment";
+    },
+  });
+}
+
+export function useContactverzoekenByKlantId(
```

Comment:
is dit een 'shared' functie?

nb. hebben we niet alle contactverzoeken meegekregen met het ophalen van de contactmomenten? neem aan dat de enige reden om ze nog eens apart op te halen de paginering is? hoeveel contactmomenten en contactverzoeken heeft een klant gemiddeld en hoe lang worden die bewaard? is overwogen om de paginering eruit te halen?

Reply:
Ik weet niet wanneer we iets als "shared" functie definiëren. Deze functie wordt momenteel maar op één plek aangeroepen. Toch zou m'n voorkeur gaan om 'm in een service te hebben, aangezien het ophalen van contactmomenten a.d.h.v. klant id w.m.b. een herbruikbare functie mag zijn.

We hebben inderdaad alle verzoeken meegekregen, dit heeft echter nog een refactor nodig. 

**Huidige werkwijze:**

- GET alle contactmomenten filter verzoeken eruit => ContactmomentenOverview;
- GET alle contactmomenten filter momenten eruit => ContactverzoekenOverview;

Hier gaat de paginering kapot (we tonen immers een fontend filtered array, maar de paginering wordt door de gateway gebaseerd op de ongefilterde array).

**Gewenste werkwijze:**

- GET enkel contactmomenten => ContactmomentenOverzicht;
- GET enkel contactverzoeken => ContactverzoekenOverzicht;

Deze filter is nog niet beschikbaar in de gateway. Dit zou wel bovenstaande issues oplossen.

Ik stel voor dit nu alvast te mergen, zodat de kern van deze functionaliteiten al wel getest kunnen worden.

Reply:
als hij niet herbruikbaar hoeft te zijn, dan liever niet in een shared map plaatsen. je kan hem in de service.ts van contactmoment plaatsen. 

Ik weet niet of het uitbreiden van de filtermogelijkheden van de gateway, hier de beste oplossing zou zijn. Wat mij betreft laten we het nu zo, maar mocht dit onderdeel er in de toekomst nog aangepast worden, dan vind ik dat we de optie om helemaal geen paginering te gebruiken serieus moeten overwegen.

Reply:
- Heb 'm uit de shared functions gehaald.
- De backend filters waren intussen geregeld, dus deze heb ik verwerkt.

Ik resolve deze daarom nu.